release-notes/5.0/preview/api-diff/preview2/Asp.Net/5.0-preview2_System.Security.Principal.md

```diff
@@ -0,0 +1,11 @@
+# System.Security.Principal
```

Comment:
Why is this in the Asp.Net  folder?

Reply:
Asp.Net ships this as a part of their ref pack

Reply:
Isn't System.Security.Principal.Windows part of netcoreapp?